erpnext_pk/install.py

```diff
@@ -0,0 +1,51 @@
+import frappe
+from frappe.custom.doctype.custom_field.custom_field import create_custom_fields
+
+
+
+custom_fields = {
+	# Doctype Company Mods
```

Comment:
What means "Mods" in the comment?

Reply:
We go by the philosophy of writing clean and elegant code. Clean enough to be understood by developers without much comments. Comments are for supporting the complex code and for delineating different parts of the code. We'll see what we can do about these